tests/zwe-remote-integration/src/__tests__/init/__snapshots__/generate.test.ts.snap

```diff
@@ -345,6 +345,16 @@ exports[`init-generate (SHORT) BAD: missing defaults.yaml 1`] = `
 ZWEL0318E - failed to read '/test/dir/files/defaults.yaml' - EDC5129I No such file or directory. (errno2=0x05620062)"
 `;
 
+exports[`init-generate (SHORT) PARMLIB negatives 1`] = `"ERROR: ZWEL0316E Invalid PARMLIB format PARMLIB(ZOWE.PR4285(A)."`;
```

Comment:
Is this right? The output message is printed without `"`, but the backtick will keep it.

Reply:
Yes, this is an artifact in how Jest captures snapshots under-the-hood using `JSON.stringify`. There's no quote in the actual command output.